todo_app/data/Card.py

```diff
@@ -0,0 +1,39 @@
+from todo_app.data.trello_urls import trello_urls
+import requests
+class Card:
+    def __init__(self,id, name, status):
+        self.id = id
+        self.name = name
+        self.status = status
```

Comment:
Currently this calls the setter each time a `Card` is instantiated, which I don't think is the intention!
```suggestion
        self._name = name
        self._status = status
```